### PR TITLE
suggested edits.

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: intro.tex
Comment (patrickingraham) [inline intro.tex:1]: I shuffled the phrasing around so it's clear from the beginning who the customers of this document are (meaning Chilean unionized employees and not AURA except)

File: content.tex
Comment (patrickingraham) [inline content.tex:16]: One concept that was in Cristian's original document, but not this one, was the concept of being on-call and receiving ~10% of the wage, but then increasing to 100% if someone actually calls them. It seems that is discussed in the allowances section but isn't introduced here?

File: content.tex
Comment (patrickingraham) [inline content.tex:25]: You mention and allowance section here and in the next section, except there doesn't appear to be an allowance section?

File: content.tex
Comment (patrickingraham) [inline content.tex:72]: I think IT specific... which means the Alerting section should probably be a subsection.

File: content.tex
Comment (patrickingraham) [inline content.tex:34]: I find this wording to be a bit confusing regarding the de-escalation and day-shift..
You're saying that work done by the day shift is considered in-hours... but it sounds like if an issue is de-escalated then the out-of-hours work are not specially compensated.
How about:

Some out of hours support tasks may being as remote support and escalate to in-person support. If during the out-of-hours support, the issues is de-escalated and no longer urgent, then the following day shift can be assigned to fix the problem as part of their standard shift.

File: content.tex
Comment (patrickingraham) [inline content.tex:32]: I think AURA needs to provide the vehicle... or at least offer one. People cannot be expected to provide their own transportation. 

Are you even allowed to drive your own vehicle up? If so, then they need to get compensated for mileage etc.

File: content.tex
Comment (patrickingraham) [inline content.tex:59]: why the TBD?

File: content.tex
Comment (patrickingraham) [inline content.tex:94]: I was struggling to understand the wording here. I think there was just a lot in one sentence. Is it correct that ALL shifts must be authorized by HR? Or just the concept of these types of shifts?

File: content.tex
Comment (aalexov) [reply content.tex:16]: The allowances section was the only place where it was originally discussed and I've kept it there.  If you think it needs to be up front, we can intro it there.  I just didn't want to repeat the calculation as it was provided later in the document.

File: content.tex
Comment (aalexov) [reply content.tex:32]: Ok, I can be more explicit about the AURA vehicle.  The person must have the approval to drive to/from the summit as a prerequisite.  I can add.

File: content.tex
Comment (aalexov) [reply content.tex:59]: The TBD was from Cristian's doc.  Not sure - would need to ask him to resolve it.

File: content.tex
Comment (aalexov) [reply content.tex:94]: My understanding is that we need to get HR authorization ONCE for the concepts and sign off on the pay.  (Who knows what happens with union discussions...).  My interpretation of this document from Cristian was that is was the basis of getting HR approval overall, so that out of hours help can happen as needed, without approval per shift. Definitely something to ask Cristian.

**Review by aalexov — APPROVED** [review]

File: content.tex
Comment (aalexov) [reply content.tex:16]: added to intro:
On-call compensation is 10\% times the hourly wage for the duration of an official on-call shift.  This increases to 100\% times the hourly wage if the person is called to work out of hours time.

File: content.tex
Comment (aalexov) [reply content.tex:72]: Alerts to On-Call Support and In-Person Emergency Support staff and can be activated by the people present at the summit during on-sky observing.  It is anticipated this will be done using a call list.

Additionally, alerts will also be triggered by the monitoring stack and routed to the incident management tool using the commodity internet.  These automated alerts will more likely be DevOps related.

In DevOps the incident management tool will control the shift rotations, escalations, and will alert according to the severity of the fault.

**Pull request merged**
